x/stakeibc/keeper/msg_server_update_inner_redemption_rate_bounds_test.go

```diff
@@ -1 +1,111 @@
-package keeper
+package keeper_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	_ "github.com/stretchr/testify/suite"
+
+	stakeibctypes "github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+type UpdateInnerRedemptionRateBoundsTestCase struct {
+	validMsg stakeibctypes.MsgUpdateInnerRedemptionRateBounds
+	zone     stakeibctypes.HostZone
+}
+
+func (s *KeeperTestSuite) SetupUpdateInnerRedemptionRateBounds() UpdateInnerRedemptionRateBoundsTestCase {
+	// Register a host zone
+	hostZone := stakeibctypes.HostZone{
+		ChainId:           HostChainId,
+		HostDenom:         Atom,
+		IbcDenom:          IbcAtom,
+		RedemptionRate:    sdk.NewDec(1.0),
+		MinRedemptionRate: sdk.NewDec(9).Quo(sdk.NewDec(10)),
+		MaxRedemptionRate: sdk.NewDec(15).Quo(sdk.NewDec(10)),
+	}
+
+	s.App.StakeibcKeeper.SetHostZone(s.Ctx, hostZone)
+
+	defaultMsg := stakeibctypes.MsgUpdateInnerRedemptionRateBounds{
+		// TODO: does this need to be the admin address?
```

Comment:
I don't think so cause I don't think validate basic is run when you call the msg server function directly